webhook_server/tests/test_custom_check_runs.py

```
@@ -0,0 +1,912 @@
+
+        mock_pull_request = Mock()
+        mock_pull_request.number = 123
+        mock_pull_request.base = Mock()
+        mock_pull_request.base.ref = "main"
+
+        # Simulate /retest custom:lint command
+        check_config = mock_github_webhook.custom_check_runs[0]
+
+        # Create async context manager mock
+        mock_checkout_cm = AsyncMock()
+        mock_checkout_cm.__aenter__ = AsyncMock(return_value=(True, "/tmp/worktree", "", ""))
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/worktree"

(S108)